packages/zarf-agent/zarf.yaml

```
@@ -19,19 +19,12 @@ components:
     required: true
     images:
       - "###ZARF_PKG_TMPL_AGENT_IMAGE_DOMAIN######ZARF_PKG_TMPL_AGENT_IMAGE###:###ZARF_PKG_TMPL_AGENT_IMAGE_TAG###"
-    manifests:
+    charts:
       - name: zarf-agent
+        releaseName: zarf-agent
```

Bit of a hack - but if we modify this - given that the raw chart name has not changed since creation and that the hasher logic is quite old as well [link](https://github.com/zarf-dev/zarf/blob/main/src/internal/packager/helm/common.go#L38-L42) then we should have quite substantial backwards compatibility AND it should function now. 
```suggestion
        releaseName: zarf-d2db14ef40305397791454e883b26fc94ad9615d
```

I say we start here to begin - test other E2E functionality while continuing to get feedback on this as the solution.